Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1480,6 +1480,15 @@ namespace xr
                     {
                         // Only signal that gamepad data is available if the actions were available
                         InputSources[idx].GamepadTrackedThisFrame = true;
```

Comment:
nit: mark gamepad as tracked after setting all the values

Reply:
Sure!